source/Pages/compute_on_spider.rst

```diff
@@ -305,7 +318,408 @@ combination that is not available you will receive the following error message:
 
    'sbatch: error: Batch job submission failed: Requested node configuration is not available'
 
+===============
+Using GPU nodes
+===============
+
+To run your program on GPU nodes some guidelines for the user have to be taken into account. Firstly, GPUs are only available on the GPU nodes ``wn-gp-[01,02]`` and ``wn-ga-[01,02]`` and **not** on the UI nodes. All GPU nodes run Nvidia hardware and as such, CUDA software is necessary. The CUDA drivers are installed on the relevant machines, but the CUDA interface and other programs need to be run in a singularity container. Nvidia has containers available on the internet for CUDA use, that can be built upon. These can be found `here <https://catalog.ngc.nvidia.com/containers>`_. Alternatively, you can build your own container from scratch, which is also shown in the next section. Before building the container, the user needs to know the version of the drivers available on the GPU nodes. The version can be found with:
+
+.. code-block:: bash
+
+   srun -p GPU_PARTITION --gpus GPU:N_GPUS nvidia-smi
+
+where the GPU_PARTITION is either ``gpu_v100`` or ``gpu_a100`` depending on which one you are planning to use. The ``--gpus`` flag specifies which type of GPU you want to use and how many, you will get ``N_GPUS`` up to the maximum in the cluster of type ``GPU`` which can be ``v100`` or ``a100``. The compilation and running of code has to be done inside of a singularity container, so start by building a singularity image. More information on singularity on :abbr:`Spider (Symbiotic Platform(s) for Interoperable Data Extraction and Redistribution)` can be found at :ref:`singularity containers <singularity-containers>`. Once the container is available, the program can be run. If container building permissions are not enabled for you on the GPU nodes, please contact us at :ref:`our helpdesk <helpdesk>`.
```

Comment:
1) "has to be done" is perhaps a bit strong. I.e., Singularity images may be our preferred way of dealing with this, but its certainly not the only way
2) "If container building" Please make clear that this concerns Singularity only via the fakeroot option. Perhaps also provide the error that the user would get in this case